patches/0002-Solve-standard-descriptors-visibility-problem.patch

```diff
@@ -0,0 +1,62 @@
+From 356f85a077adfcb4a1c66163b7d3a2c38a1667b3 Mon Sep 17 00:00:00 2001
+From: Marius-Cristian Baciu <2309bmcristi@gmail.com>
+Date: Sat, 18 Jul 2020 23:58:35 +0300
+Subject: [PATCH] Solve standard descriptors visibility problem
+
+initStdio() is a routine we have defined in D runtime lib's gluecode.c
+---
```

Comment:
Signed-off-by is missing, try adding it here "by hand".
```suggestion
initStdio() is a routine we have defined in D runtime lib's gluecode.c

Signed-off-by: Marius-Cristian Baciu <2309bmcristi@gmail.com>
---
```

Reply:
Added Signed-off-by